pkg/point/response.go

```diff
@@ -0,0 +1,57 @@
+package point
+
+type CreateResponse struct {
+	Amount         int                   `json:"amount"`
+	ID             string                `json:"id"`
+	Description    string                `json:"description"`
+	State          string                `json:"state"`
+	DeviceID       string                `json:"device_id"`
+	Payment        CreatePaymentResponse `json:"payment"`
+	AdditionalInfo AdditionalInfo        `json:"additional_info"`
```

Comment:
```suggestion
	DeviceID       string                `json:"device_id"`
	
	Payment        CreatePaymentResponse `json:"payment"`
	AdditionalInfo AdditionalInfo        `json:"additional_info"`
```

Reply:
Nao entendi pq separar a struct por com espaço, qual a melhoria?

Reply:
estamos seguindo esse estilo na medida do possível
https://github.com/uber-go/guide/blob/master/style.md#embedding-in-structs

Reply:
Ajustado!!